configs/errors.tslint.json

```diff
@@ -38,6 +38,51 @@
     "no-any": true,
     "no-consecutive-blank-lines": true,
     "no-construct": true,
+    "no-implicit-dependencies": {
+      "options": [
+        true,
+        "dev",
+        [
+          "@phosphor/algorithm",
+          "@phosphor/commands",
+          "@phosphor/coreutils",
+          "@phosphor/domutils",
+          "@phosphor/dragdrop",
+          "@phosphor/messaging",
+          "@phosphor/properties",
+          "@phosphor/signaling",
+          "@phosphor/virtualdom",
+          "@phosphor/widgets",
+          "chai",
+          "chai-string",
+          "electron",
+          "express",
+          "fs-extra",
+          "inversify",
+          "jsdom",
+          "lodash.debounce",
+          "lodash.throttle",
+          "monaco-languageclient",
+          "native-keymap",
+          "nsfw",
+          "react",
+          "react-autosize-textarea",
```

Comment:
strange that we have it in core, but it is used only in scm, maybe we better just to declare it scm for now

Reply:
I'll make the changes :)